lib/active_admin/dependency.rb

```diff
@@ -55,6 +55,10 @@ def self.[](name)
       Matcher.new name.to_s
     end
 
+    def self.rails5?
+      rails >= '5.beta'
```

Comment:
I think the correct match is `5.0.0.beta`
![rails___rubygems_org___your_community_gem_host](https://cloud.githubusercontent.com/assets/165599/12880761/fb2f6422-ce43-11e5-8a2a-e28b84b11020.jpg)